client/src/views/workspaces/WorkspacesPage.vue

```diff
@@ -232,11 +232,17 @@ onMounted(async (): Promise<void> => {
     displayView.value = savedData.displayState;
   }
 
-  hotkeys = hotkeyManager.newHotkeys(Groups.Workspaces);
-  hotkeys.add('n', Modifiers.Ctrl, Platforms.Desktop, openCreateWorkspaceModal);
-  hotkeys.add('g', Modifiers.Ctrl, Platforms.Desktop, async () => {
-    displayView.value = displayView.value === DisplayState.Grid ? DisplayState.List : DisplayState.Grid;
-  });
+  hotkeys = hotkeyManager.newHotkeys();
+  hotkeys.add(
+    { key: 'n', modifiers: Modifiers.Ctrl, platforms: Platforms.Desktop, disableIfModal: true, route: Routes.Workspaces },
+    openCreateWorkspaceModal,
+  );
+  hotkeys.add(
+    { key: 'g', modifiers: Modifiers.Ctrl, platforms: Platforms.Desktop, disableIfModal: true, route: Routes.Workspaces },
```

Comment:
Hotkey disabled after visiting `UsersPage` then back with the header button (not sidebar)